test/env.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
   notifyApiKey: 'NOTIFY TEST, TEAM OR LIVE API KEY STRING',
   notifyEmailTemplateId: 'NOTIFY EMAIL TEMPLATE STRING',
   // https://docs.notifications.service.gov.uk/node.html#smoke-testing
-  notifyEmailAddress: 'simulate-delivered@notifications.service.gov.uk'
+  notifyEmailAddress: 'simulate-delivered@notifications.service.gov.uk',
+  notifyEmailAddresses: 'simulate-delivered@notifications.service.gov.uk,simulate-delivered@notifications.service.gov.uk'
```

Comment:
does this need to be an array as test is failing in `send-emails.test`. See below comment.

`notifyEmailAddresses: ['simulate-delivered@notifications.service.gov.uk', 'simulate-delivered@notifications.service.gov.uk']`

Reply:
This was originally how I defined this. However, whenever you define an environment var in these file there are parsed to string, i.e. `notifyEmailAddresses: ['simulate-delivered@notifications.service.gov.uk', 'simulate-delivered@notifications.service.gov.uk']` would equal `'['simulate-delivered@notifications.service.gov.uk', 'simulate-delivered@notifications.service.gov.uk']'`